_posts/2025-04-28-towards-more-rigorous-llm-evals.md

```diff
@@ -283,10 +294,12 @@ Looking at the example template (Figure 1 from the paper, reproduced above), we
 
 In other words, the original GSM8K question cannot be generated from the proposed symbolic template.
 A more appropriate sampling range for `ans` might be $(5, 100)$ (i.e. the same as $x$, $y$ and $z$), and for `total` might be $(10, 100)$.
+<!-- Perhaps mention again Fig 1, because I was confused about which x, y, and z you are talking about.  -->
 
 
 As we saw in Section 4.1.1, the accuracy of both models decreases as the number of digits increases, indicating that larger inputs are harder to process.
-We use the logistic regression model from that section try quantify the difference in accuracy that might arise from using the ranges in the paper vs those we propose here (the "reasoning" that gets us to the correct mathematical expression is correct.)
+We use the logistic regression model from that section to quantify the difference in accuracy that might arise from using the ranges in the paper vs those we propose here (the "reasoning" that gets us to the correct mathematical expression is correct.)
+<!-- Bit unclear what you want to say here, especially what's in the brackets. -->
```

Comment:
that bit was still very much under development; I've changed it quite a bit since then and going through the exercise of actually computing something that makes sense made me realise the values I proposed initially aren't good enough either. 

This really makes me believe the authors did a very bad job at creating those templates, given that the one they choose to show is so flawed...